client/my-sites/stats/overview.jsx

```diff
@@ -88,21 +88,22 @@ export default React.createClass( {
 			);
 
 			return overview;
-		}, this );
+		} );
 
 		return (
-			<Main wideLayout={ true }>
+			<Main wideLayout>
+				<QuerySites allSites />
```

Comment:
We should not be using `QuerySites allSites` anywhere, but I see there's a few cases in Jetpack already? cc @johnHackworth